src/pages/jdform.tsx

```diff
@@ -0,0 +1,81 @@
+import type { NextPage } from 'next';
+import { useForm, SubmitHandler } from 'react-hook-form';
+import axios from 'axios';
+import { useRouter } from 'next/router';
+
+interface IJdForm {
+  description: String;
+}
+
+const JdForm: NextPage = () => {
+  const router = useRouter();
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<IJdForm>();
+
+  const onSubmit: SubmitHandler<IJdForm> = (data) => {
+    const value = localStorage.getItem('jobLisId');
+
+    axios
+      .post('api/v1/job-descriptions', {
+        rawText: data.description,
+      })
+      .then(function (response) {
+        console.log(response);
+        if (typeof value === 'string') {
+          const id = JSON.parse(value);
+          console.log(response.data.id);
+          axios
+            .patch(`api/v1/job-listings/${id}`, {
+              jobDescriptionId: response.data.id,
+            })
+            .then(function (response) {
+              console.log(response);
+              router.push('/applications');
+            })
+            .catch(function (error) {
+              console.log(error);
+            });
```

Comment:
We might be able to skip this step if we modify the backend endpoint.

I'll need to double check!

Reply:
Looks like this was fixed in a later commit. Well done!

Reply:
My intial aim was to how can I complete the task so did few api request. Somehow I manage to make it work. Next I watched some videos from Prisma team to make it more efficient and reduce endpoint calls.